homeassistant/components/madvr/sensor.py

```diff
@@ -185,14 +185,14 @@ class MadvrSensorEntityDescription(SensorEntityDescription):
         value_fn=lambda coordinator: coordinator.data.get(OUTGOING_COLOR_SPACE),
         translation_key=OUTGOING_COLOR_SPACE,
         device_class=SensorDeviceClass.ENUM,
-        options=["RGB", "444", "422", "420"],
+        options=["RGB", "444", "422", "420", "?"],
```

Comment:
Instead of a question mark make the entity return `None` as native_value so it renders as `unknown`

Reply:
updated to do this in native_value